src/main/java/servlets/FetchOptionsServlet.java

```diff
@@ -61,7 +61,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       string in the given array is already capitalized, since this is the case
       for search strings, in which all Strings are fully upper-cased, and
       this servlet is being used to fetch those search strings. */
-  public ArrayList<String> titleCaseItems(ArrayList<String> arr) {
+  public static ArrayList<String> titleCaseItems(ArrayList<String> arr) {
```

Comment:
Is there a restriction on the strings to contain only alphabets?

Reply:
No